partcad-cli/src/partcad_cli/click/commands/add/sketch.py

```
@@ -0,0 +1,55 @@
+@click.argument(
+    "kind",
+    type=click.Choice(
+        [
+            "cadquery",
+            "build123d",
+            "dxf",
+            "svg",
+            "basic",
+            "ai-cadquery",
+            "ai-openscad",
+        ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)